_gsocproposals/2020/proposal_IntAlertSystem.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Intelligent Alert Management System For HEP experiments
+layout: gsoc_proposal
+project: Monitoring
+year: 2020
+organization: CERN
+---
+
+## Description
+
+We would like to build an intelligent Alert Management System for HEP experiments.
+It should embrace existing open-source solutions based on Prometheus and AlertManager
+to select alerts from various sub-systems and reduce amount
+of non-relevant alerts. The system should be able to correlate alerts from
+CERN MONIT infrastructure based on ElasticSearch, Kafka, Grafana stack, GGUS WLCG, and
+Prometheus, AlertManager, VictoriaMetrics software components used in HEP experiments.
```

Comment:
I'm missing also an explanation of what are the current solutions and their shortcomings